bodhi/tests/client/test___init__.py

```diff
@@ -1475,7 +1475,7 @@ def test_url_flag(self, send_request):
         result = runner.invoke(
             client.create_release,
             ['--name', 'F27', '--url', 'http://localhost:6543', '--username', 'bowlofeggs',
-             '--password', 's3kr3t'])
+             '--password', 's3kr3t', '--critpath-pkgs', 'python,perl,php'])
```

Comment:
This test is here to test proper operation of the ```--url``` flag, so let's not complicate it by inserting this flag.